src/components/InMoreLanguageSwitch.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div class="wikibase-termbox-subsection-switch">
+		<div class="wikibase-termbox-subsection-switch__switch">
+			<a :href="link" @click="clientShowAllLanguages( $event )">{{ message( MESSAGE_KEYS.ALL_LANG ) }}</a>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { VueConstructor } from 'vue';
+import Component, { mixins } from 'vue-class-component';
+import Messages, { MessagesMixin } from './mixins/Messages';
+
+@Component
+export default class TermBox extends ( mixins( Messages ) as VueConstructor<MessagesMixin> ) {
+	public readonly link: string;
+	constructor() {
+		super();
+		this.link = 'place-holder-link';
+	}
+
+	public clientShowAllLanguages( event: MouseEvent ) {
```

Comment:
This might only ever be triggered in the client, but I don't that it has "client" in the name, especially since it doesn't need to get an event as an input.

Reply:
done